load_screen_fails_and_caregivers_into_screen_follow_up.R

```diff
@@ -63,6 +63,14 @@ rc_upload <- screen_fails_and_caregivers %>%
 
 # specify local context for data upload
 #load_dot_env('local.env')
-redcap_write(rc_upload,
+result <- redcap_write(rc_upload,
         redcap_uri = Sys.getenv("URI"),
         token = Sys.getenv("SCREEN_TOKEN"))
+
+message <- paste0("Uploaded rows: ", nrow(rc_upload),
+        "\nAffected records: ", result$records_affected_count
+        )
+
+email_subject <- paste("Screen fails loaded", Sys.getenv("INSTANCE"), script_run_time, "| Success", write_data$success)
```

Comment:
```suggestion
email_subject <- paste("Screen fails loaded", Sys.getenv("INSTANCE"), script_run_time, "| Success", result$success)
```

You had the wrong variable for the results of the `redcap_write`

Reply:
Stole code from myself and didn't respect the variable names